### PR TITLE
Move tests agnostic to Windows OS from Windows lane

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:

These test doesn't require Windows OS to be
tested. CI has only one copy of Windows license
and therefore limited bw on Windows lane.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by maiqueb — APPROVED** [review]
Makes sense to me.

File: tests/hyperv_test.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/pkg/controller"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/libvmi"
+	"kubevirt.io/kubevirt/tests/util"
```
Comment (maiqueb) [inline tests/hyperv_test.go:20]: Please separate the import groups.

Comment (dhiller) [reply tests/hyperv_test.go:20]: @maiqueb @xpivarc  `make generate` should take care of this no?

Comment (maiqueb) [reply tests/hyperv_test.go:20]: I think so. It'll at least sort them (can't promise the import groups will make sense..).

**Comment by maiqueb** [comment]
/lgtm cancel

I was too quick w/ the approve button - didn't notice you're adding a test but not running `make generate`.

**Review by maiqueb — APPROVED** [review]
Thank you.

**Comment by xpivarc** [comment]
@maiqueb PTAL

**Comment by xpivarc** [comment]
/retest
@maiqueb :ping_pong:

**Review by maiqueb — APPROVED** [review]
Thanks for the PR; sorry I didn't reply sooner.

**Pull request merged**